### PR TITLE
[CS] Add inference page @open sesame 09/24 19:32

### LINKED ISSUE #571: [CS/EDJE] Change router to add inference
Change router to contain inference pages

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Change home to accomodate inference
- Route draw inference
- Add `draw_target` to use appdata to address label

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

resolves #571 

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

File: Applications/Tizen_native/CustomShortcut/inc/data.h [hunk 1]
```diff
@@ -37,6 +37,13 @@
 
 #define FEATURE_SIZE 62720
 #define NUM_CLASS 2
+
+typedef enum _DRAW_TARGET {
```
Comment (kparichay) [inline Applications/Tizen_native/CustomShortcut/inc/data.h:41]: _DRAW_TARGET -> DRAW_TARGET_

Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/inc/data.h]: Done

**Comment by zhoonit** [comment]
v2. Fix missing license.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**